tests/integration/ha_tests/helpers.py

```diff
@@ -334,3 +341,109 @@ async def get_units_hostnames(ops_test: OpsTest) -> List[str]:
         f"{unit.name.replace('/', '-')}.mongodb-k8s-endpoints"
         for unit in ops_test.model.applications[APP_NAME].units
     ]
+
+
+async def db_step_down(ops_test: OpsTest, sigterm_time: datetime) -> None:
+    """Pipes the k8s logs, looking for stepdown message."""
+    kubectl_cmd = [
+        "microk8s",
+        "kubectl",
```

Comment:
Should most probably check if we have regular kubectl, so that microk8s isn't a hard dependency for the tests.

Reply:
true, but the tests run on a microk8s cloud (i.e. microk8s is already installed) and I don't think we will ever change from that considering Canonicals love for microk8s.